ql-qlf-k4n8-plugin/synth_quicklogic.cc

```diff
@@ -139,6 +139,8 @@ struct SynthQuickLogicPass : public ScriptPass {
 
     void script() override
     {
+        static const std::string opt_opts = "-nosdff -nodffe";
```

Comment:
I don't believe this it the right solution.  There is a pass that will convert unsupported FF's into other FF's.  Let me find the pass.

Reply:
I believe `dfflibmap` is the one of the relevant passes.

Reply:
The other option is `dffsr2dff`

Reply:
`dfflegalize`?

Reply:
> `dfflegalize`?

Ya, that one!

Reply:
Ok, I'll change that to use `dfflegalize`

Reply:
For clarity, the DFF behavior change is relatively recent.  `dfflegalize` is a new pass as a result of the changes.

Reply:
the flow now uses `dfflegalize`